app/javascript/hyacinth_v1/components/projects/enabled_dynamic_fields/EnabledDynamicFieldForm.js

```diff
@@ -287,6 +294,10 @@ export const EnabledDynamicFieldForm = (props) => {
       .then(historyPromise);
   };
 
+  const onSuccessHandler = (result) => {
```

Comment:
Good idea adding this!  No reason to keep the user on the page after they hit save.